.github/workflows/maven.yml

```diff
@@ -32,3 +32,22 @@ jobs:
           restore-keys: ${{ runner.os }}-m2
       - name: Build
         run: mvn ${{ env.maven_commands }}
+  deploy:
+    if: contains('refs/heads/develop', github.ref)
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Repository
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+          server-id: ome.snapshots
```

Comment:
Is this going to work for non-SNAPSHOTs? I expect they should be deployed to `ome.staging`?

Reply:
I think I need to revisit that i.e. change the server target if it is a tag.

Reply:
Main downside of this approach is that in our current workflow there will be two events associated with a commit with a non-SNAPSHOT version: the push and the tag. If we handle the tag, the push event will still fail I think when deploying to `ome.snapshots`.

https://github.com/actions/setup-java/pull/86 might be useful in our case but it's not integrated yet.

Reply:
When a commit is pushed to develop,  I extract the version out of the pom.xml.
If the version contains ``SNAPSHOT`` -> ``ome.snapshots`` is used,  otherwise``ome.staging`` will be used.